multiversx_sdk_core/transaction.py

```diff
@@ -60,6 +67,12 @@ def to_dictionary(self, with_signature: bool = True) -> Dict[str, Any]:
         dictionary["receiver"] = self.receiver.bech32()
         dictionary["sender"] = self.sender.bech32()
 
+        if self.sender_username:
```

Comment:
Correct, that's is the ordering within `FrontendTransaction`, the one used for signatures.

https://github.com/multiversx/mx-chain-core-go/blob/main/data/transaction/frontendTransaction.go